tests/unit/test_mock_build.py

```diff
@@ -0,0 +1,6 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+def test_build_in_mock_default_resultdir():
+    result = run_packit_command(["build", "in-mock"])
```

Comment:
Not sure where did you get this from, cause I can't find this function anywhere… but I see `call_real_packit()`…

1. Switch to that function
2. Pass `cwd` too
3. And also, `build in-mock` requires a config for the environment it should be built in, that gets passed via `-r ‹target›`, I'm not really sure what would be the most suitable target to be used here…